tests/python-pytest/onnx/onnx_test.py

```diff
@@ -106,6 +106,16 @@ def test_squeeze():
     output = mxnet_backend.run_node(node_def, [input1])[0]
     npt.assert_almost_equal(output, np.squeeze(input1, axis=[1, 3]))
 
+@with_seed()
```

Comment:
can we not use the backend tests added  to the fw here: https://github.com/onnx/onnx/pull/497/files

Reply:
Yes, i can enable that. I will wait for https://github.com/apache/incubator-mxnet/pull/10605 to merge and then add appropriately ( as it modifies  test  files)

Reply:
Enabled ONNX backend test for Unsqueeze